server/helpers/express-utils.ts

```diff
@@ -125,6 +126,34 @@ function getCountVideos (req: express.Request) {
   return req.query.skipCount !== true
 }
 
+// helpers added in server.ts and used in subsequent controllers used
+const apiResponseHelpers = (req, res: express.Response, next = null) => {
```

Comment:
I don't think it should be a middleware. Why don't you create an helper function `sendError (res: express.Response)` instead?

Reply:
Why? It seems more logical to replace response middlewares with a response middleware.